pyproject.toml

```diff
@@ -2,6 +2,12 @@
 build-backend = "hatchling.build"
 requires = ["hatch-vcs", "hatchling"]
 
+[dependency-groups]
+dev = [
+  "mne[dev]",
+  "PySide6 != 6.7.0, != 6.8.0, != 6.8.0.1",
```

Comment:
didn't we have issues in interactivity with PySide6?

Reply:
Not sure, but I'm happy to change it to `PyQt6` if there are issues with `PySide6`. I added the latter because it is still the official Qt bindings package.